paper-world-defaults.yml

```diff
@@ -303,11 +262,12 @@ max-growth-height:
   reeds: 3
 misc:
   disable-end-credits: false
-  disable-relative-projectile-velocity: false
+  disable-relative-projectile-velocity: true
```

Comment:
Instructs the server to ignore shooter velocity when calculating the velocity of a fired arrow.